doc/cci_guide.md

```diff
@@ -0,0 +1,1703 @@
+# 1. Test Objective
+The cci test is aimed to test CUBRID CCI driver. We write cci test cases to test all kinds of CCI Applications, such as `cci_connect()`, `cci_connect_with_url()`, `cci_cursor()`, `cci_fetch()`, `cci_get_data()`, `cci_get_result_info()` and so on. Actually cci test is a branch of shell test, the only difference is that cci shell scripts must execute C codes which include CUBRID CCI API.
+
+
+# 2. Execute CCI Test
```

Comment:
Refactor categories as below:

2.1 Install CUBRID that we plan to test


2.2 Prepare test cases
Generally, we check out test cases from git repository. CCI test cases are located in https://github.com/CUBRID/cubrid-testcases-private/tree/develop/interface/CCI/shell.

```
.....
```

2.3 Install CTP
Please refer to the guide to [install CTP in Linux platform]( https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#1-install-ctp-in-linux-platform).

Create cci test configuration file as below:
File CTP/conf/cci.conf
```
scenario=$HOME/cubrid-testcases-private/interface/CCI/shell/_20_cci
```

2.4 Execute test
```
ctp.sh shell -c ./cci.conf
```

2.5 Examine test results

Reply:
install cubrid, we can  use run_cubrid_install in CTP,I'll install CTP> install CUBRID